ImageStreamIO.c

```diff
@@ -349,20 +353,34 @@ int ImageStreamIO_typesize(
 {
     switch (datatype)
     {
-    case _DATATYPE_UINT8:         return SIZEOF_DATATYPE_UINT8;
-    case _DATATYPE_INT8:          return SIZEOF_DATATYPE_INT8;
-    case _DATATYPE_UINT16:        return SIZEOF_DATATYPE_UINT16;
-    case _DATATYPE_INT16:         return SIZEOF_DATATYPE_INT16;
-    case _DATATYPE_UINT32:        return SIZEOF_DATATYPE_UINT32;
-    case _DATATYPE_INT32:         return SIZEOF_DATATYPE_INT32;
-    case _DATATYPE_UINT64:        return SIZEOF_DATATYPE_UINT64;
-    case _DATATYPE_INT64:         return SIZEOF_DATATYPE_INT64;
-    case _DATATYPE_HALF:          return SIZEOF_DATATYPE_HALF;
-    case _DATATYPE_FLOAT:         return SIZEOF_DATATYPE_FLOAT;
-    case _DATATYPE_DOUBLE:        return SIZEOF_DATATYPE_DOUBLE;
-    case _DATATYPE_COMPLEX_FLOAT: return SIZEOF_DATATYPE_COMPLEX_FLOAT;
-    case _DATATYPE_COMPLEX_DOUBLE:return SIZEOF_DATATYPE_COMPLEX_DOUBLE;
-    default:                      break;
+    case _DATATYPE_UINT8:
+        return SIZEOF_DATATYPE_UINT8;
+    case _DATATYPE_INT8:
+        return SIZEOF_DATATYPE_INT8;
+    case _DATATYPE_UINT16:
+        return SIZEOF_DATATYPE_UINT16;
+    case _DATATYPE_INT16:
+        return SIZEOF_DATATYPE_INT16;
+    case _DATATYPE_UINT32:
+        return SIZEOF_DATATYPE_UINT32;
+    case _DATATYPE_INT32:
+        return SIZEOF_DATATYPE_INT32;
+    case _DATATYPE_UINT64:
+        return SIZEOF_DATATYPE_UINT64;
+    case _DATATYPE_INT64:
+        return SIZEOF_DATATYPE_INT64;
+    case _DATATYPE_HALF:
+        return SIZEOF_DATATYPE_HALF;
+    case _DATATYPE_FLOAT:
+        return SIZEOF_DATATYPE_FLOAT;
+    case _DATATYPE_DOUBLE:
+        return SIZEOF_DATATYPE_DOUBLE;
+    case _DATATYPE_COMPLEX_FLOAT:
+        return SIZEOF_DATATYPE_COMPLEX_FLOAT;
+    case _DATATYPE_COMPLEX_DOUBLE:
+        return SIZEOF_DATATYPE_COMPLEX_DOUBLE;
+    default:
+        break;
```

Comment:
This difficult-to-read style is probably what caused an error in a previous commit.

Just sayin'